content/zh-cn/docs/contribute/localization_zh.md

```diff
@@ -349,6 +349,10 @@ Website 的仓库中 `scripts/linkchecker.py` 是一个工具，可用来检查
 - annotation，注解
 - anti-affinity，反亲和性
 - attach，挂接
+- authenticate，身份验证
+- authorizer，鉴权器
+  - 其动词 authorize，可译为 “授权”
+  - 其名词 authorization，可译为 “授权”、“鉴权”
```

Comment:
```suggestion
  - 其动词 authorize，译为 “授权”
  - 其名词 authorization，译为 “授权”、“鉴权”
```

If a selective description is used, it will lose the meaning of discussion.

Reply:
authorize 确实在不同上下文中有“鉴权”和“授权”两种含义，相对还比较容易区分。
如果表达的是将权限授予某主体，译为“授权”会更合适，如果表达的是根据某些参数条件来确定主体应该具有哪些权限，这一操作译为“鉴权”更合适。

Reply:
ok, slightly changed, including the conditional branch of 授权 and 鉴权.